crates/query-group-macro/src/queries.rs

```diff
@@ -48,7 +48,8 @@ impl ToTokens for TrackedQuery {
                 quote!(#(#options),*)
             })
             .into_iter()
-            .chain(self.lru.map(|lru| quote!(lru = #lru)));
+            .chain(self.lru.map(|lru| quote!(lru = #lru)))
+            .chain(Some(quote!(unsafe(non_update_return_type))));
```

Comment:
That should be changed before merge, right?

Reply:
Sure, if someone can point me to how to fix it.